packages/teams-js/src/public/meeting.ts

```diff
@@ -189,6 +189,9 @@ export namespace meeting {
 
   function getIncomingClientAudioStateHelper(): Promise<boolean> {
     return new Promise<boolean>(resolve => {
+      if (!isSupported()) {
```

Comment:
We are converting meeting.ts to v1 form. Do we still want these changes?

Reply:
I think Lakhveer is right. We should close this review (sorry for not deciding this earlier!).